README.md

```diff
@@ -1 +1,14 @@
-Orbit
+# Exposure System
+
+### Description
+
+Exposure system is a new initiative to expose **Humans of AltCampus** to various different knowledge other than just programming, by providing them with articles and tasks on a regular basis.
+
+This system also aims at bringing the people out of their comfort zone by giving them challenges, which would be in pairs or in groups. Completion of each of these challenges will result in positive incentives, while not participating will result in disincentives, motivating people to keep carrying on.
+
+### Steps to get started
```

Comment:
@Yjspoder @AmanSharma4419 Steps should be a bit detailed. For example, you have missed - `npm install`.

Reply:
@prank7 Resolved